jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/utils/Modal.java

```diff
@@ -0,0 +1,101 @@
+package com.epam.jdi.light.material.elements.utils;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.complex.ISetup;
+import com.epam.jdi.light.elements.interfaces.base.HasClick;
+import com.epam.jdi.light.material.annotations.JDIModal;
+import com.epam.jdi.light.material.asserts.utils.ModalAssert;
+
+import java.lang.reflect.Field;
+
+import static com.epam.jdi.light.elements.pageobjects.annotations.objects.FillFromAnnotationRules.fieldHasAnnotation;
+
+public class Modal extends UIBaseElement<ModalAssert> implements HasClick, ISetup {
+
+    private static final String BUTTON = "#root > div > button";
+    private static final String SIMPLE_MODAL_TITLE = "#simple-modal-title";
+    private static final String SIMPLE_MODAL_DESCRIPTION = "#simple-modal-description";
+    private static final String TRANSITION_MODAL_TITLE = "#transition-modal-title";
+    private static final String TRANSITION_MODAL_DESCRIPTION = "#transition-modal-description";
+    private static final String SERVER_MODAL_TITLE = "#server-modal-title";
+    private static final String SERVER_MODAL_DESCRIPTION = "#server-modal-description";
```

Comment:
Please delete all these strings and use annotations fields in methods. Also, create two methods for verifying description and  title instead of getSimpleModalTitle(), getTransitionModalTitle(), getServerModalTitle(), getSimpleModalDescription(), getTransitionModalDescription(), getServerModalDescription(). The same thing for verifySimpleModalTitle(), verifyTransitionModalTitle(), verifyServerModalTitle() and etc